build/ci_utils/src/extensions/command.rs

```diff
@@ -15,7 +15,7 @@ pub trait CommandExt {
             let _ = write!(ret, "\n\twith working directory: {}", cwd.display());
         };
         let env = self.as_std().get_envs();
-        if !env.is_empty() {
+        if env.len() != 0 {
```

Comment:
When I was using rust last time, every structure with `len` was strongly encouraged by linter to have also `is_emptry`. Has something changed?

Reply:
That `is_empty` was unstable, part of `exact_size_is_empty` feature.

Reply:
So, rust does not follow its own rules :)

Reply:
It's not an official rust rule, but a clippy configuration. Also it only checks things that actually have stable `is_empty` method, so this change doesn't trigger a warning.